assets/js/modules/idea-hub/datastore/idea-state.js

```diff
@@ -30,7 +30,7 @@ import Data from 'googlesitekit-data';
 import { createFetchStore } from '../../../googlesitekit/data/create-fetch-store';
 import { actions as errorStoreActions } from '../../../googlesitekit/data/create-error-store';
 import { actions as moduleDataActions } from './module-data';
-const { receiveError, clearError } = errorStoreActions;
+const { receiveError, clearErrors } = errorStoreActions;
```

Comment:
I think we still need to use `clearErrors` instead of `clearError`, right?

Reply:
@kuasha420 we need to revert all changes in this file because `createFetchStore` already takes care of errors for the `updateIdeaState` base name. Other custom errors like `saveIdea`, `unsaveIdea` and `dismissIdea` should stay intact.

Reply:
So, the first bullet point from the IB is not needed at all?

Reply:
Fully reverting the file doesn't give the desired behavior of the error message disappearing on tab switch. Can you take a look please? 

> Other custom errors like saveIdea, unsaveIdea and dismissIdea should stay intact.

I think at least we need to change these to `updateIdeaState` for the AC to be satisfied? Not sure about the `clearErrors` vs `clearError` though. 

Maybe @asvinb and @tofumatt (they worked on the IB) can chime in here as well.

Cheers!

Reply:
@kuasha420 see here: https://github.com/google/site-kit-wp/pull/4290#discussion_r741168418